charts/asset-manager/templates/freshclam-cronjob.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ .Release.Name }}-freshclam
+spec:
+  schedule: "0 * * * *"
```

Comment:
Just to be nice neighbours, I'd probably set the minute to something randomly-chosen that isn't 0. Unfortunately it looks like k8s CronJob still doesn't do fuzzing of start times.

Reply:
(I'm assuming we're not planning on running more than like 3 of these, right? If not then we'll need to go a bit further, say by adding a random starting delay inside the job.)

Reply:
I also believe there is a rate limit on how often you can poll also

Reply:
ok, maybe I will change the architecture. I will have only one freshdaemon per cluster and the volume will be mounted in the needed containers.

I need to do some more checks on PVs for our cluster thanks.